invisible_cities/reco/sensor_functions.py

```diff
@@ -7,11 +7,28 @@
 from ..sierpe             import fee as FE
 from .                    import wfm_functions as wfm
 
+
 def convert_channel_id_to_IC_id(data_frame, channel_ids):
     return pd.Index(data_frame.ChannelID).get_indexer(channel_ids)
 
+def charge_fluctuation(signal, single_pe_rms):
+    """Simulate the fluctuation of the pe before noise addition
+    produced by each photoelectron
+    """
+    if single_pe_rms == 0:
+        ## Protection for some versions of numpy etc
+        return signal
+    else:
+        ## We need to convert to float to get accuracy here
+        sig_fl = signal.astype(float)
+        sigma = np.sqrt(sig_fl[sig_fl>0]) * single_pe_rms
```

Comment:
assign `sig_fl > 0` to a variable and surround the `>` and `<` signs with spaces.